new-pipeline/style/dashboards.css

```diff
@@ -126,10 +126,15 @@
 .mg-marker-text { font-size: 12px !important; }
 .mg-markers line { stroke-width: 2px !important; }
 
-#dist-caption {
+#dist-caption, #measure-description {
+  margin-left: 2em;
+  white-space: nowrap;
+  width : 55em;
```

Comment:
Do we need to limit the width?
It would be great if we could just use the full available page width for the description text.

Reply:
Okay ! Will do !